build-scripts/for-changelog/svn-changelog.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/sh
```

Comment:
This script is necessary for generating the changelog from SVN and is not required for build at github. I'd removed it from the PR